src/masonite/notification/Notification.py

```diff
@@ -0,0 +1,51 @@
+"""Base Notification facade."""
+from abc import ABC, abstractmethod
```

Comment:
I think we can remove this. Although having contract classes are great I haven't actually seen them be very useful. Especially because Python is not typed and this only ensures the methods are on the class and not necessarily that the signature of those methods match. So its like a half-ass contract class .. I just haven't seen them be useful at all. I haven't added them on any other features and I haven't ported them over from M3 .. 

I'm open for conversation on this but yeah I don't see it being a useful thing to add

Reply:
Okay I'll remove